packages/components/src/renderer/custom.tsx

```diff
@@ -9,9 +9,10 @@ const checkError = (moduleInjector: Function, component: string, payload: string
   if (!moduleInjector) {
     return new Error(`${errorPrepend} could not get module injector`)
   }
-  if (!module) {
-    return new Error(`${errorPrepend} "module" is not defined in the payload`)
-  }
+  // TODO: what is this?
+  // if (!module) {
+  //   return new Error(`${errorPrepend} "module" is not defined in the payload`)
+  // }
```

Comment:
Any ideas?

Reply:
I think `payload` should be renamed to `module` in the function's args as it would make much more sense (@charlescatta maybe you can add details here?)

But if I understand this piece of code correctly, when rendering custom content-types, we want to import the renderer from a certain module using the `InjectedModuleView`. This is the reason why we want to check if the module is specified, otherwise, it would fail to render.